docs/LearningOutcomes.adoc

```diff
@@ -6,6 +6,7 @@
 :stylesDir: stylesheets
 ifdef::env-github[]
 :note-caption: :information_source:
+:site-githuburl: ../
```

Comment:
Don't we need to go a bit deeper?

From `https://github.com/se-edu/addressbook-level1/blob/master/docs/LearningOutcomes.adoc`, we will need to go `../../../` to reach `https://github.com/se-edu/addressbook-level1`